src/components/Product.jsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
-import { useAddProduct } from '../recoil/hooks';
+import { useAddProduct, useRemoveProduct } from '../recoil/hooks';
 
 const Product = ({ product }) => {
   const addProduct = useAddProduct();
+  const removeProduct = useRemoveProduct();
   
   return (
     <div style={{ display: 'flex', flexDirection: 'column' }}>
       <p>{product.title}</p>
       <p>{product.price}</p>
       <button onClick={() => addProduct(product)}>Add to cart</button>
+      <button onClick={() => removeProduct(product)}>Remove from cart</button>
```

Comment:
removeProduct(product.id)

You can change the hook to receive only the Id of the product, you'll need only this to remove it from the list

Reply:
Oh, that would be better, I'll make the PR to update the product quantity and change this, thanks for the feedback @luansantosti!

Reply:
Great, I'll merge this PR then! o/